nrich-form-configuration/README.md

```diff
@@ -30,12 +34,13 @@ To be able to use this library following configuration is required:
         formIdConstraintHolderMap.put("testRequest.formId", FormConfigurationServiceTestRequest.class);
 
         return new DefaultFormConfigurationService(validator.getValidator(), formIdConstraintHolderMap, constrainedPropertyValidatorConverterServiceList);
-     }
+    }
 
     @Bean
     public FormConfigurationController formConfigurationController(FormConfigurationService formConfigurationService) {
         return new FormConfigurationController(formConfigurationService);
     }
+}
```

Comment:
On line 49, it seems to me that `FormConfigurationServiceNestedTestRequest` class name should be used instead of `MessageSourceFieldErrorMessageResolverServiceTestRequest`?

On line 73, there is a typoo in `On client side REST API enpoint`.